### PR TITLE
3.6 groups table kyselov

### PR DESCRIPTION
**Pull request by IselDrew**
Add groups model and migration files

### CODE REVIEW

File: src/modules/groups/groups.model.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import { DataTypes, Model, Sequelize } from 'sequelize';
+
+export interface IGroups {
```
Comment (Alan18081) [inline src/modules/groups/groups.model.ts:3]: Why do we need this interface?

File: src/modules/groups/groups.model.ts [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  id?: number;
+  group_name: string;
```
Comment (Alan18081) [inline src/modules/groups/groups.model.ts:5]: Use camel-case for field names

File: src/modules/groups/groups.model.ts [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+      {
+        id: {
+          type: DataTypes.INTEGER,
+          primaryKey: true,
+          allowNull: false,
+          autoIncrement: true,
+        },
+        group_name: new DataTypes.STRING(),
+        group_token: new DataTypes.STRING(255),
+        teacher_id: new DataTypes.INTEGER(),
```
Comment (Alan18081) [inline src/modules/groups/groups.model.ts:33]: For each column define the same set of rules as you have defined in migration

File: src/migrations/20200415143345-create-group.ts [hunk 4]
```diff
@@ -0,0 +1,47 @@
+import { QueryInterface, DataTypes } from 'sequelize';
+import { Groups } from '../modules/groups/groups.model';
+
+export async function up(query: QueryInterface) {
+  try {
```
Comment (Alan18081) [inline src/migrations/20200415143345-create-group.ts:5]: We don't need try ... catch blocks here. Just return the calling of createTable method

File: src/migrations/20200415143345-create-group.ts [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+        comment: 'Date of the last update',
+      },
+    });
+  } catch (e) {
+    return Promise.reject(e);
+  }
+}
+
+export async function down(query: QueryInterface) {
+  try {
```
Comment (Alan18081) [inline src/migrations/20200415143345-create-group.ts:42]: We don't need try ... catch blocks here. Just return the calling of dropTable method

File: src/modules/groups/groups.model.ts (hunk 1 above)
Comment (IselDrew) [reply src/modules/groups/groups.model.ts]: removed

File: src/modules/groups/groups.model.ts (hunk 2 above)
Comment (IselDrew) [reply src/modules/groups/groups.model.ts]: fixed

File: src/modules/groups/groups.model.ts (hunk 3 above)
Comment (IselDrew) [reply src/modules/groups/groups.model.ts]: fixed

File: src/migrations/20200415143345-create-group.ts (hunk 4 above)
Comment (IselDrew) [reply src/migrations/20200415143345-create-group.ts]: removed

File: src/migrations/20200415143345-create-group.ts (hunk 5 above)
Comment (IselDrew) [reply src/migrations/20200415143345-create-group.ts]: removed

**Pull request merged**